build_tools/third_party/cuda/BUILD.template

```diff
@@ -37,6 +37,7 @@ cc_library(
     hdrs = glob([
         "include/**/*.h",
         "include/**/*.hpp",
+        "include/nv/**/*",
```

Comment:
I currently have `v11.6` installed and I see a few other files that _could_ be included under `include/cuda/**`
![image](https://user-images.githubusercontent.com/4010439/232152818-07151cdf-29a7-426f-a2a9-99850ce4b1db.png)


1) Is just adding `include/nv/**/*` enough?
2) What files in that directory are useful to include and use? Are there specific errors that you see without them listed here?

Reply:
The error I was seeing when building `openxla-pjrt-plugin` was
```
ERROR: /home/pjannaty/.cache/bazel/_bazel_pjannaty/20c4e80672769c9136b56ef6e532bdcb/external/iree_core/runtime/src/iree/hal/drivers/cuda/BUILD.bazel:15:24: Compiling runtime/src/iree/hal/drivers/cuda/nop_executable_cache.c failed: (Exit 1): clang failed: error executing command (from target @iree_core//runtime/src/iree/hal/drivers/cuda:cuda) /usr/lib/llvm-12/bin/clang -U_FORTIFY_SOURCE -fstack-protector -Wall -Wthread-safety -Wself-assign -Wno-free-nonheap-object -fcolor-diagnostics -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' ... (remaining 122 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
In file included from external/iree_core/runtime/src/iree/hal/drivers/cuda/nop_executable_cache.c:7:
In file included from external/iree_core/runtime/src/iree/hal/drivers/cuda/nop_executable_cache.h:12:
In file included from external/iree_core/runtime/src/iree/hal/drivers/cuda/context_wrapper.h:11:
In file included from external/iree_core/runtime/src/iree/hal/drivers/cuda/cuda_headers.h:11:
In file included from bazel-out/k8-opt/bin/external/nccl/_virtual_includes/headers/third_party/nccl/nccl.h:11:
In file included from external/iree_cuda/include/cuda_fp16.h:4006:
external/iree_cuda/include/cuda_fp16.hpp:65:10: fatal error: 'nv/target' file not found
#include <nv/target>
         ^~~~~~~~~~~
1 error generated.
```

see [Discord](https://discord.com/channels/689900678990135345/1095829553466843208) thread for context.

With only adding `include/nv` I manage to build fine now. However, there are whole bunch of directories with files without extensions under `cuda/include` now. I can also do a blanket whitelisting (`include/**/*`) if you prefer.

Reply:
Seems reasonable to me to have the entire `include/**/*` path. Well, I'd say that before seeing that there are extensionless files there... but I can try to forget that and remain blissfully ignorant :P

Reply:
Just close your eyes and pretend they're not there...

Reply:
Aaah why don't they have extensions though :cry: